testnet/stacks-node/src/config.rs

```diff
@@ -1085,6 +1106,25 @@ impl NodeConfig {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct MinerConfig {
+    pub min_tx_fee: u64,
+    pub min_cumulative_tx_fee: u64,
+    pub first_attempt_time_ms: u64,
+    pub subsequent_attempt_time_ms: u64,
+}
```

Comment:
Can you document these new config options somewhere in this repo? Does the README.md make sense?

Reply:
Is there any mining documentation in the repo anywhere I can add to?

Reply:
Added a section to the README